crates/counter-agg/src/lib.rs

```diff
@@ -276,14 +281,14 @@ impl MetricSummary {
         Ok(Some(result_val))
     }
 
-    pub fn prometheus_rate(&self) -> Result<Option<f64>, CounterError>{
-        let delta  = self.prometheus_delta()?;
+    pub fn prometheus_rate(&self) -> Result<Option<f64>, CounterError> {
+        let delta = self.prometheus_delta()?;
         if delta.is_none() {
             return Ok(None);
         }
         let delta = delta.unwrap();
-        let bounds = self.bounds.unwrap() ; // if we got through delta without error then we have bounds
-        /* bounds stores [L,H), but Prom takes the duration using the inclusive range [L, H-1ms]. So subtract an extra ms from the duration*/
+        let bounds = self.bounds.unwrap(); // if we got through delta without error then we have bounds
+                                           /* bounds stores [L,H), but Prom takes the duration using the inclusive range [L, H-1ms]. So subtract an extra ms from the duration*/
```

Comment:
I don't understand why this whitespace space was added? I think this comment refers to the line below; does making this a `//` comment help keep it aligned?

Reply:
Swapping to `//` didnt' keep it aligned but moving it to the end of the line that its a comment for did